man/dracut.cmdline.7.asc

```diff
@@ -1171,9 +1171,15 @@ rd.live.overlay=/dev/sdb1:persistent-overlay.img
 rd.live.overlay=UUID=99440c1f-8daa-41bf-b965-b7240a8996f4
 --
 
-**rd.live.overlay.cowfs=**__[btrfs|ext4|xfs]__::
-Specifies the filesystem to use when formatting the overlay partition.
-The default is ext4.
+**rd.live.autooverlay=**__<devspec>__::
+Specifies the path to a device on which to create a new partition
+for persistent storage.
++
+* *_rd.live.autooverlay_* takes precedence over *_rd.live.overlay_* at boot.
+
+**rd.live.autooverlay.cowfs=**__[f2fs|btrfs|ext4|xfs]__::
+Specifies the filesystem to use when formatting the autooverlay partition.
+The default is f2fs.
```

Comment:
Please keep the filesystem list in alphabetical order, and set `ext4` as the default.

Reply:
Done.